shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -325,10 +326,15 @@ public void resize(
                 physicalHeight,
                 () -> {
                   unlockInputConnection(vdController);
+                  // Converting back to logic pixels requires a context, which may no longer be
```

Comment:
Sounds like the only way this fails is if the user changes the scale/UI density in the settings app while the Flutter app is running, right?

Reply:
That seems like a reasonable tradeoff vs crashing

Reply:
I think so, and they'd have to do it *during this race*, which seems pretty unlikely.

And I'm still not sure what the repro for this is; it may well be that it only happens during teardown and the return value will be ignored anyway.